  Reason: Fix segmentation fault with gftp_expand_path(src=0x0) and null pointer
  Source: Gentoo
Upstream: gftp is unmaintained, gftp-evolved will come out some day

diff --git a/lib/misc.c b/lib/misc.c
index 16c019b..2791466 100644
--- a/lib/misc.c
+++ b/lib/misc.c
@@ -143,6 +143,8 @@ gftp_expand_path (gftp_request * request, const char *src)
        tempchar;
   struct passwd *pw;
 
+  g_return_val_if_fail(src != NULL, NULL);
+
   pw = NULL;
   str = g_strdup (src);
 
